src/app/components/ContactEmailSettingsModal.js

```diff
@@ -37,6 +39,31 @@ const PGP_MAP = {
     [SEND_PGP_MIME]: PGP_MIME
 };
 
+const keyComparator = (originalKeys = [], trustedFingerprints = []) => (firstKey, secondKey) => {
+    const firstKeyFingerprint = firstKey.getFingerprint();
+    const secondKeyFingerprint = secondKey.getFingerprint();
+    const isFirstKeyTrusted = trustedFingerprints.includes(firstKeyFingerprint);
+    const isSecondKeyTrusted = trustedFingerprints.includes(secondKeyFingerprint);
+    if (isFirstKeyTrusted ^ isSecondKeyTrusted) {
+        // the trusted key takes preference
+        return isFirstKeyTrusted ? -1 : +1;
```

Comment:
`+1` is useless, right?
```suggestion
        return isFirstKeyTrusted ? -1 : 1;
```

Reply:
haha, yeah, that's because that's how I would hand-write it :sweat_smile: